docs/source/user_guide/dataset_creation/datasets.rst

```diff
@@ -5865,3 +6047,44 @@ corresponding to the type of dataset that you are working with.
     Note that, as this type represents a labeled video dataset, its importer
     must be a subclass of |LabeledVideoDatasetImporter|, and its exporter must
     be a subclass of |LabeledVideoDatasetExporter|.
+
+  .. group-tab:: Grouped datasets
+
+    The pseudocode below provides a template for a custom |GroupDatasetType|
+    subclass:
+
+    .. code-block:: python
+        :linenos:
+
+        import fiftyone.types as fot
+
+        class CustomGroupDataset(fot.GroupDataset):
+            """Custom grouped dataset type."""
+
+            def get_dataset_importer_cls(self):
+                """Returns the
+                :class:`fiftyone.utils.data.importers.GroupDatasetImporter`
+                class for importing datasets of this type from disk.
+
+                Returns:
+                    a :class:`fiftyone.utils.data.importers.GroupDatasetImporter`
+                    class
+                """
+                # Return your custom GroupDatasetImporter class here
+                pass
+
+            def get_dataset_exporter_cls(self):
+                """Returns the
+                :class:`fiftyone.utils.data.exporters.GroupDatasetExporter`
+                class for exporting datasets of this type to disk.
+
+                Returns:
+                    a :class:`fiftyone.utils.data.exporters.GroupDatasetExporter`
+                    class
+                """
+                # Return your custom GroupDatasetExporter class here
+                pass
+
+    Note that, as this type represents an unlabeled image dataset, its importer
```

Comment:
Is this correct? Seems like an odd statement

Reply:
Good catch! Copy paste error 😿